src/main.ts

```diff
@@ -256,7 +290,7 @@ const handlePushToSourceBranch = async ({
 	return {
 		baseBranch: newPr.base.ref,
 		headBranch: newPr.head.ref,
-		sourceBranch: originalHead,
+		sourceBranch: pushedBranch,
 		targetBranch,
 		url: newPr.html_url,
 	};
```

Comment:
I forgot to check if we need to keep returning. If the PR already existed and we didn't merge to it, I don't think we should return it. We didn't touch it.

Reply:
Got these changes in